### PR TITLE
docs(ses): corrected default value as true for ConfigurationSet.reputationMetrics property

### LINKED ISSUE #30968: (aws-ses): ConfigurationSet.reputationMetrics incorrect default value
### Describe the issue

```ses.ConfigurationSet.reputationMetrics``` states the default value for this property is ```false``` whereas in reality it defaults to ```true```.

### Links

https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ses.ConfigurationSet.html#reputationmetrics

### PR DESCRIPTION
**Pull request by ashishdhingra**
### Issue # (if applicable)

Closes #30968.

### Reason for this change
The default value of `ConfigurationSet.reputationMetrics` if not set in the CDK stack is actually `true` (refer https://github.com/aws/aws-cdk/issues/30968#issuecomment-2257161637 for details).

### Description of changes
Corrected default value as `true` for `ConfigurationSet.reputationMetrics` property in documentation comment.

### Description of how you validated changes
N/A

### Checklist
- [x] My code adheres to the [CONTRIBUTING GUIDE](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md) and [DESIGN GUIDELINES](https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md)

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Comment by pahud** [comment]
Thank you. Based on your verification on the console https://github.com/aws/aws-cdk/issues/30968#issuecomment-2257161637 leaving it undefined would make it enabled by CFN so I think this PR should be good.

LGTM but I'll let the maintainer to approve.

**Review by Leo10Gama — APPROVED** [review]
Confirmed on my end as well that it defaults to true in the console, thank you!

**Pull request merged**
